_pages/en_US/finalizing-setup.txt

```diff
@@ -131,17 +131,18 @@ If, before following this guide, you already had an EmuNAND setup and would like
   + This process will take some time
   + If you get an error, make sure that you have at least 1.3GB of free space on your SD card
 1. Press (A) to continue
-1. Press (B) to return to the main menu
-1. Select "Exit"
-1. Press (A) to relock write permissions if prompted
-1. Navigate to `[M:] MEMORY VIRTUAL`
-1. Press (A) on `boot9.bin` to select it
-1. Select "Copy to 0:/gm9/out"
+1. Select "Dump Options"
+1. Select "Dump Boot9.bin & Boot11.bin"
+1. Press (A) to confirm
+1. Press (A) to continue
+1. Select "Dump OTP.bin"
+1. Press (A) to confirm
 1. Press (A) to continue
+1. Press (B) to return to the main menu
 1. Press (Home) to bring up the action menu
 1. Select "Poweroff system" to power off your device
 1. Insert your SD card into your computer
-1. Copy `<date>_<serialnumber>_sysnand_###.bin`, `<date>_<serialnumber>_sysnand_###.bin.sha`, `essential.exefs`, and `boot9.bin` from the `/gm9/out/` folder on your SD card to a safe location on your computer
+1. Copy `<date>_<serialnumber>_sysnand_###.bin`, `<date>_<serialnumber>_sysnand_###.bin.sha`, `essential.exefs`, `boot9.bin`, 'boot11.bin' and 'OTP.bin' from the `/gm9/out/` folder on your SD card to a safe location on your computer
```

Comment:
Same as above, `OTP.bin` is unnecessary.

As well, rename `boot9.bin` and `boot11.bin` to include `<serialnumber>_` in front, so that it is consistent with what is actually being dumped, as well being consistent with the guide's formatting